examples/crd_api.rs

```diff
@@ -23,8 +23,11 @@ use kube::{
 #[kube(printcolumn = r#"{"name":"Team", "jsonPath": ".spec.metadata.team", "type": "string"}"#)]
 pub struct FooSpec {
     #[validate(length(min = 3))]
```

Comment:
let's change this (and similar ones) to `#[schemars(length(min = 3))]`

Reply:
ah, nice. I should've dug into schemars' docs a bit more, this makes more sense.